keyboards/smallkeyboard/keymaps/via/keymap.c

```diff
@@ -0,0 +1,28 @@
+/* Copyright 2021 zhouqiong19840119
```

Comment:
This keymap folder needs a `rules.mk` with:
```makefile
VIA_ENABLE = yes
```

Reply:
THANKS sir